docs/types.rst

```diff
@@ -813,9 +813,14 @@ for each ``_KeyType``, recursively.
             balances[msg.sender] = newBalance;
         }
     }
+  
+::
+
+    pragma solidity ^0.4.0;
 
     contract MappingUser {
-        address contractAddress = 0x42;
+        // the address of a MappingExample contract
+        address contractAddress = 0x42...;
```

Comment:
Thanks @axic for the follow up. Indeed the Interface would be needed.

What about using `new MappingExample().balances(this)` instead of `MappingExample(address).balances(this)`?